docs/source/install.rst

```diff
@@ -13,9 +13,12 @@ Using Pipenv
 
         $ pipenv install aiogram
 
-Using AUR
+Using Pacman
 ---------
-*aiogram* is also available in Arch User Repository, so you can install this framework on any Arch-based distribution like ArchLinux, Antergos, Manjaro, etc. To do this, use your favorite AUR-helper and install the `python-aiogram <https://aur.archlinux.org/packages/python-aiogram/>`_ package.
+*aiogram* is also available in Arch Linux Repository, so you can install this framework on any Arch-based distribution like Arch Linux, Antergos, Manjaro, etc. To do this, just use pacman to install the `python-aiogram <https://archlinux.org/packages/community/any/python-aiogram/>`_ package:
+    .. code-block:: bash
```

Comment:
@felixonmars Could you put a newline before this line? Otherwise the text block above is (somehow) rendered bold

![image](https://user-images.githubusercontent.com/17182159/104291086-ef581e80-54dc-11eb-87af-27ecb1ad368e.png)

```suggestion

    .. code-block:: bash
```

Reply:
Sure, updated now :)